xml/rtsp_servers.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fingerprints matches="rtsp_header.server" protocol="rtsp" database_type="service">
```

Comment:
Please add `preference="0.85"` which will put it under `http_banners.xml` in priority.

Reply:
Done.